zuul.d/edpm_build_images.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- job:
```

Comment:
In check, We want to pass containers(including qcow)  to bmaas job. I wonder if changing this job to content provider would make more sense. We can discuss this further.

Reply:
@bshewale As discussed in today's design for the check we want to create a content-provider job, Please modify this job following the existing content-provider job pattern.

Reply:
We decided to do this in a followup